Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -87,6 +89,10 @@ public struct WorldScaleSceneView: View {
                         locationDataSource: locationDataSource,
                         sceneView: sceneViewBuilder
                     )
+                    .onSingleTapGesture { tapPoint in
+                        let scenePoint = arScreenToLocation(screenPoint: tapPoint)
+                        onSingeTapGestureAction?(tapPoint, scenePoint)
+                    }
```

Comment:
These 4 blocks that define the GeoTrackingSceneView and WorldTrackingSceneView can all be put into a function or computed property and called instead of duplicating this code.